tests/acceptance/common.py

```diff
@@ -282,6 +282,9 @@ def common_board_setup(conn, files=None, remote_path="/tmp", image_file=None):
 
 
 def common_board_cleanup(conn):
+    # Get the coverage files from the client before the device is cleaned up
+    get_no_sftp("/data/mender/coverage*", conn, local=".")
```

Comment:
I think this is only used by physical boards, which has not been used for ages and is probably buggy beyond belief by now. You should probably add this to `qemu_finalizer` instead.